prog/postgres/postgres_resource_nexus.rb

```diff
@@ -148,6 +156,7 @@ def before_run
 
     decr_destroy
 
+    strand.children.each { _1.destroy }
```

Comment:
It needs a rebase
https://github.com/ubicloud/ubicloud/pull/993/files

Reply:
That is in PostgresServer, this is PostgresResource. We need this in both, because both can bud a prog and switch to destroy before it is completed.

Reply:
Sorry, my bad. I was curious about how this line not conflicted with main 😄